templates/candidates-office.html

```diff
@@ -0,0 +1,34 @@
+{% extends 'layouts/main.html' %}
+
+{% block title %}
+  Browse candidates
+{% endblock %}
+
+{% block body %}
+
+<header class="page-header slab slab--primary">
+  <h1 class="page-header__title tst-page-title">Candidates</h1>
```

Comment:
We're not actually using this `tst-page-title` class for testing on this page are we? Might as well remove it.